src/database/database.ts

```diff
@@ -0,0 +1,28 @@
+import { createClient } from '@supabase/supabase-js'
+import { SupabaseClient } from '@supabase/supabase-js/src'
+import { GenericSchema } from '@supabase/supabase-js/src/lib/types'
+
+class Database {
+  private static supabaseUrl = 'https://yipnhkmklmbuxjwzpipg.supabase.co'
+  private static key: string =
+    process.env.SUPABASE_ANON_KEY != null ? process.env.SUPABASE_ANON_KEY : ''
+  supabaseKey =
```

Comment:
Remove key from code and add as environment variable (see comment above)

Reply:
Generate new key, this one is already on GitHub public